README.md

```diff
@@ -14,6 +14,21 @@ The script can install, remove, and check the status of an installed Salt minion
 either using a direct command line option or via VMware's use of Guest Variables,
 commonly referred to as guestVars.
 
+In every two-step installation example, you would be well-served to **verify against the SHA256
+sum** of the downloaded ``svtminion.sh`` file.
+
+.. _sha256sums:
+
+The SHA256 sum of the ``svtminion.sh`` file, per release, is:
```

Comment:
```suggestion
The SHA256 sum of the `svtminion.sh` file, per release, is:
```

Updating from `rST` to Markdown.

Reply:
will do, reckoned on some changes from rst to md